homeworks/olha.tymchyshyn_olyatmchshn/5-logic/styles.css

```diff
@@ -0,0 +1,60 @@
+.calculator-container {
+  display: flex;
```

Comment:
Any interactions with UI elements should not lead to jumping off the layout
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/fd572e56-bc92-4831-9ce4-521a99870204)
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/5414c868-6221-46e9-89ae-f4beb9a070e9)